src/client/java/daos-java/src/main/native/io_daos_dfs_DaosFsClient.c

```diff
@@ -829,32 +898,31 @@ Java_io_daos_dfs_DaosFsClient_dfsReadDir(JNIEnv *env, jobject client,
 		nr = READ_DIR_BATCH_SIZE;
 		rc = dfs_readdir(dfs, dir, &anchor, &nr, entries);
 		if (rc) {
-			char *tmp = "Failed to read %d more entries from " \
-					"directory after reading %d " \
+			char *tmp = "Failed to read %d more entries from "
+					"directory after reading %d "
 					"entries.\n buffer length: %d";
-			char *msg = (char *)malloc(strlen(tmp) + 4 + 4 + 4);
+			char *msg = NULL;
 
-			sprintf(msg, tmp, READ_DIR_BATCH_SIZE, total, size);
+			asprintf(&msg, tmp, READ_DIR_BATCH_SIZE, total, size);
 			throw_exception(env, msg, rc);
 			failed = 1;
 			break;
 		}
 		if (!nr) continue;
 		total += nr;
 		int i;
-		for(i=0; i<nr; i++){
+		for(i = 0; i < nr; i++){
```

Comment:
(style)  space required before the open brace '{'
(style)  space required before the open parenthesis '('